spec/core/UtilSpec.js

```diff
@@ -31,6 +31,38 @@ describe("jasmineUnderTest.util", function() {
     });
   });
 
+  describe("isPromise", function () {
+
+    it("should return true when passed a native promise", function () {
+      if (typeof Promise === 'undefined') { return; }
```

Comment:
Let's use the `requirePromises` function supplied by Jasmine's promise helpers, for a bit more visibility on when this test is being skipped.